ajax/libs/vertx/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "vertx",
-  "filename": "vertxbus.min.js",
-  "version": "2.0.0",
+  "filename": "vertx-eventbus.min.js",
+  "version": "3.3.3",
   "description": "Effortless asynchronous application development for the modern web and enterprise. This library internally uses SockJS to send and receive data to a SockJS vert.x server called the SockJS bridge. http://vertx.io/core_manual_js.html#sockjs-eventbus-bridge",
   "homepage": "http://vertx.io/",
   "licenses": [
-    "EPL-1.0",
-    "Apache-2.0"
+    "Apache-2.0",
+    "EPL-1.0"
```

Comment:
Any reason to change the license order here? I noticed that you removed and added EPL-1.0 back, and you should keep the minimum diff, the diff here is meaningless.